client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -65,9 +66,10 @@ export default class ProductFormDetailsCard extends Component {
 					</FormLabel>
 				</div>
 				<div className="products__product-form-details-wrapper">
-					<div className="products__product-form-details-images">
-
-					</div>
+					<ProductFormImages
+						product={ product }
+						editProduct={ editProduct }
+					/>
```

Comment:
I haven't handled this one yet since we don't currently have a way to disable the save button / flag that a blocking operation is occurring. I can open an issue so we can loop back.